### PR TITLE
step10 Sort Tasks by Creation Date in Descending Order

### PR DESCRIPTION
**Pull request by fanfannb**
Overview
This PR modifies the task list to display tasks in descending order based on their creation date, rather than by their ID as previously. This change improves the user experience by showing the most recent tasks first.

Changes
Tasks Controller Update: Updated the index action in the TasksController to fetch tasks ordered by created_at in descending order.
View Adjustments: Minor adjustments to the index view to support and highlight the new sorting behavior.
Justification
Displaying tasks in descending order of creation makes it easier for users to find the most recently added tasks, which are often the most relevant to current work. This change aligns with common user expectations for task management systems.

截屏2024-06-03 15 24 27

### ステップ10: タスク一覧を作成日時の順番で並び替えましょう ★ スキップ可能

- 現在IDの順で並んでいますが、これを作成日時の降順で並び替えてみましょう
- 並び替えがうまく行っていることをsystem specで書いてみましょう

### CODE REVIEW

File: myapp/spec/system/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  it 'displays tasks in descending order of creation' do
+    older_task = Task.create!(title: 'Older', created_at: 1.day.ago)
+    newer_task = Task.create!(title: 'Newer', created_at: 1.hour.ago)
+
+    visit tasks_path
+
+    expect(page).to have_selector('h5.card-title', text: 'Title: Newer', order: :first)
+    expect(page).to have_selector('h5.card-title', text: 'Title: Older', order: :second)
```
Comment (higuching) [inline myapp/spec/system/index_spec.rb]: if you reverse the order, will it doesn't passes the test?
Please try using a method that extracts regular expressions or elements to see the order.

**Review by kanipan — APPROVED** [review]

**Comment by kanipan** [comment]
LGTM

寿司ゆき:smile

**Review by higuching — COMMENTED** [review]
If you implement system spec of another controller, the name will be duplicated, so please create a task directory or change the spec name.😀

**Comment by fanfannb** [comment]
higuchi san pointed out that the tests for the task list ordering were wrong.     Specifically, the tests were not correctly validating the order in which tasks appeared on the page.

Based on the feedback received, several adjustments were made to both the code and the tests

Modified the existing system tests
I tested it by extracting h5.card-title by map and checking the order.

**Review by higuching — APPROVED** [review]
Thank you for your comment!
LGTM

**Pull request merged**